docs/local-dev/cli-commands.md

```diff
@@ -277,6 +278,44 @@ Options:
   -h, --help       display help for command
 ```
 
+## create
+
+Scope: `root`
+
+The create command opens up an interactive guide for you to create new things in
+your app. If you do not pass in any options it is completely interactive, but it
+is possible pre-select what you want to create using the `--select` flag, and
+provide options using `--options`, for example:
+
+```bash
+backstage-cli create --select plugin --option id=foo
+```
+
+This command is typically added as script in the root `package.json` to be
+executed with `yarn backstage-create`, using options that are appropriate for
+the organization that owns the app repo. For example you may have it set up like
+this:
+
+```json
+{
+  "scripts": {
+    "backstage-create": "backstage-cli create --scope internal --no-private --npm-registry https://acme.org/npm"
+  }
+}
+```
+
+```text
+Usage: backstage-cli create [options]
+
+Options:
+  --select <name>          Select the thing you want to be creating upfront
+  --option <name>=<value>  Pre-fill options for the creation process (default: [])
+  --scope <scope>          The scope to use for new packages
+  --npm-registry <URL>     The package registry to use for new packages
+  --no-private             Do not mark new packages as private
```

Comment:
I guess these last three aren't options because they are "global" in a sense. But it could be considered to make them options I'd say, especially if it starts happening that they don't apply to some of the commands.

Reply:
Don't want them to shop up as user input ever, that's why they're here. Although it could potentially be some kind of options that are ignored unless passed in as args. Gets a bit tricky to work with and discover those though, so feeling that this is an alright middle ground. If we end up templates that don't create packages these would simply be ignored